lib/hets/hets_error_process.rb

```diff
@@ -0,0 +1,48 @@
+module Hets
+  class HetsErrorProcess
+    HETS_ERROR_CODES = %w(400, 422)
+
+    attr_reader :error, :response
+
+    def initialize(error)
+      @error = error
+      @response = error.last_response
+    end
+
+    def handle
+      if is_hets_error?
+        process_error
+      else
+        raise Hets::NotAHetsError.new('Error was not produced due to hets.')
```

Comment:
Style/RaiseArgs: Provide an exception class and message as arguments to `raise`.